src/renderers/react/ElementProvider.tsx

```diff
@@ -102,10 +102,10 @@ export class ElementProvider<
     return (
       <ElementWrapper {...this.state.commands}>
         {this.props.consumer(
-          this.state.fieldValues,
```

Comment:
Maybe we should pass this as an object instead of arguments?

Reply:
As mentioned in PR description I was a bit torn ... yeah, let's do it.